JHUGenerator/main.F90

```diff
@@ -575,6 +576,7 @@ SUBROUTINE GetCommandlineArgs()
     call ReadCommandLineArgument(arg, "PChannel", success, PChannel, tosave=tosave)
     call ReadCommandLineArgument(arg, "VH_PC", success, VH_PC, tosave=tosave)
     call ReadCommandLineArgument(arg, "alpha_dip", success, alpha_dip, tosave=tosave)
+    call ReadCommandLineArgument(arg, "sign_of_W", success, sign_of_W, tosave=tosave)
```

Comment:
My only issue is with adding this option. It makes backward compatibility impossible because this is a major operational change tat shifts the generation from generating W+H and W-H in one job to having two separate jobs. The fact that it is defaulted to 'n' without requiring downstream that an option must be specified would also cause more trouble for the users, even incorrectly generating only W-H while their intention is to generate W- and W+H together, as it used to be in the past.

Is it possible to remove this option and implement the related changes as part of the optimization of the phase space for (i,j) incoming states? You can find some example implementations in the per-parton VBF phase space, which uses a hash (but in your case, the hash is relatively trivial).